src/main/kotlin/com/personio/synthetics/config/BrowserTestConfig.kt

```diff
@@ -39,6 +44,30 @@ fun BrowserTest.testFrequency(frequency: Duration) = apply {
     options.tickEvery = frequency.inWholeSeconds
 }
 
+/**
+ * Sets the advanced scheduling configuration for the synthetic browser test
+ * @param timeframes Time range and days when the tests should be scheduled to run
+ * - from/to time -> pass LocalTime data type values where From value is earlier than To value
+ * - days -> pass comma separated DayOfWeek data type values
+ * @param timezone Timezone where the tests should be scheduled to run
+ * ZoneId data type value automatically detected by browser is used by default
```

Comment:
I think better to use  `zoneId: offset value for the timezone, value detected  automatically by browser`

Reply:
Fixed